pysyun/conversation/flow/persistent_telegram_bot.py

```diff
@@ -1,14 +1,27 @@
-from pysyun.conversation.flow.dialog_state_machine import DialogStateMachineBuilder
 from telegram.ext import PicklePersistence, Application
+
+from pysyun.conversation.flow.dialog_state_machine import DialogStateMachineBuilder
 from pysyun.conversation.flow.telegram_bot import TelegramBot
+from pysyun.conversation.workers.scheduler import Scheduler
 
 
 class PersistentTelegramBot(TelegramBot):
 
-    def __init__(self, token, initial_state="/start", persistence_file="persistent_data.pickle"):
+    def __init__(self, token, scheduler: Scheduler = None,
+                 initial_state="/start", persistence_file="persistent_data.pickle"):
         self.application = Application \
             .builder() \
             .token(token) \
             .persistence(PicklePersistence(filepath=persistence_file)) \
             .build()
         self.state_machine = self.build_state_machine(DialogStateMachineBuilder(initial_state=initial_state)).build()
+        self.user_data = {}
```

Comment:
This is also available through the Telegram bot state. So, we cannot copy that in our code.